### PR TITLE
pull in API. render to DOM.

### PR DESCRIPTION
**Pull request by hmorri32**
shakas

### CODE REVIEW

File: lib/components/Main.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Main extends Component {
   render() {
     return (
       <div>
-        <Weathrly source='http://weatherly-api.herokuapp.com/api/weather' />
+        <Weathrly source='http://api.wunderground.com/api/395c3eaa19684a65/forecast/q/' />
```
Comment (hmorri32) [inline lib/components/Main.js:8]: Weather underground API. COOL stuff.

File: lib/components/WeatherCards.js [hunk 2]
```diff
@@ -1,30 +1,32 @@
 import React from 'react';
 
-const WeatherCards = ({ weather }) => {
-  if(!weather.length){
+const WeatherCards = ({ weatherText, weatherSimple, city }) => {
+  if(!weatherText.forecastday) {
```
Comment (hmorri32) [inline lib/components/WeatherCards.js:4]: destructuring out weatherText forecast and weatherSimple forecast. Useful for daily weather summary

File: lib/components/WeatherCards.js [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
     return (
       <div>
-        Please enter a proper location
+        please whatevers
```
Comment (hmorri32) [inline lib/components/WeatherCards.js:7]: WE should change this.

File: lib/components/Main.js (hunk 1 above)
Comment (Obleo33) [reply lib/components/Main.js:8]: Really cool!

File: lib/components/WeatherCards.js (hunk 3 above)
Comment (Obleo33) [reply lib/components/WeatherCards.js:7]: I agree. good for a placeholder

File: lib/components/WeatherCards.js (hunk 2 above)
Comment (Obleo33) [reply lib/components/WeatherCards.js:4]: this should make things a bit easier

**Pull request merged**